src/dimr_bmi.c

```diff
@@ -265,9 +277,61 @@ int update(double dt) {
 }
 
 int get_var_shape(char *key, int dims[6]) { // dims -> int[6]
```

Comment:
Even better to use 
#define DIMR_BMI_MAXDIMS (6) // Used in get_var_shape.
from dimr_bmi.h, but that's not blocking for me.